tests/vision_tests/runVisionTests.py

```diff
@@ -307,6 +317,14 @@ def strip_libtree_addresses(lib_tree):
     print("\nERROR: Vision Test Number of Frames not in range - [1 - 10000]\n")
     exit()
 
+if not 1 <= width <= 7680:
```

Comment:
Can we accommodate portrait mode -- w <= 4320 & h <= 7680 ?

Reply:
> Can we accommodate portrait mode -- w <= 4320 & h <= 7680 ?

yes, we can. I updated the PR to take images up to 7680x7680. So any image sizes within this range should be fine.